### ISSUE TITLE
Add color suggestion options for WCAG 1.0 and vision deficiency

### ISSUE THREAD

**Issue by pthorson** [Tanaguru/Contrast-Finder#9]
The Contrast Finder could be improved if it added options for WCAG 1.0 and for other color deficiencies such as color blindness. 

The http://www.dasplankton.de/ContrastA/ site calculates color deficiencies and has a link to the algorithm they use.

**Comment by mfaure** [comment]
Hi @pthorson ,

Contrast-Finder already [computes contrast against WCAG 2.0](https://github.com/Tanaguru/Contrast-Finder/wiki/Howto-use#user-content-the-ratio). Do you have a specific need for WCAG 1 ?

**Comment by pthorson** [comment]
I find that some colors which pass the WCAG 2.0 test fail the color difference test in WCAG 1.0. It is probably a nice to have as compared to the color blindness and other deficiencies testing..
